source/mainnet/net/guides/create-identity.rst

```diff
@@ -88,7 +88,7 @@ To learn more about identities and accounts, see :ref:`Identities <reference-id-
 
    .. Note::
 
-      It is no longer possible for users of |mw-gen1| on an Android device to create new identities.
+      It is no longer possible for users of |mw-gen1| to create new identities.
```

Comment:
Sam here, do we still need the full description below the note, if it is not possible to create an identity anyway?